tests/core/pyspec/eth2spec/test/phase0/fork_choice/test_get_head.py

```diff
@@ -103,17 +105,21 @@ def test_split_tie_breaker_no_attestations(spec, state):
         }
     })
 
-    # block at slot 1
+    # Create block at slot 1
     block_1_state = genesis_state.copy()
     block_1 = build_empty_block_for_next_slot(spec, block_1_state)
     signed_block_1 = state_transition_and_sign_block(spec, block_1_state, block_1)
-    yield from tick_and_add_block(spec, store, signed_block_1, test_steps)
 
-    # additional block at slot 1
+    # Create additional block at slot 1
     block_2_state = genesis_state.copy()
     block_2 = build_empty_block_for_next_slot(spec, block_2_state)
     block_2.body.graffiti = b'\x42' * 32
     signed_block_2 = state_transition_and_sign_block(spec, block_2_state, block_2)
+
+    # Tick time past slot 1 so proposer score boost does not apply
+    spec.on_tick(store, store.genesis_time + (block_2.slot + 1) * spec.config.SECONDS_PER_SLOT)
+
+    yield from tick_and_add_block(spec, store, signed_block_1, test_steps)
     yield from tick_and_add_block(spec, store, signed_block_2, test_steps)
```

Comment:
```suggestion
    time = store.genesis_time + (block_2.slot + 1) * spec.config.SECONDS_PER_SLOT
    on_tick_and_append_step(spec, store, time, test_steps)

    yield from add_block(spec, store, signed_block_1, test_steps)
    yield from add_block(spec, store, signed_block_2, test_steps)
```

(Not sure if you want to tick when adding `signed_block_2` or not)